src/Microsoft.ML/LearningPipeline.cs

```diff
@@ -26,25 +26,115 @@ public ScorerPipelineStep(Var<IDataView> data, Var<ITransformModel> model)
         public Var<ITransformModel> Model { get; }
     }
 
+
+    /// <summary>
+    /// LearningPipeline class is used to define the steps needed to perform desired machine learning task.<para/>
+    /// The steps are defined by adding a data loader (e.g. <see cref="TextLoader"/>) followed by zero or more transforms (e.g. <see cref="Microsoft.ML.Transforms.TextFeaturizer"/>) 
+    /// and atmost one trainer/learner (e.g. <see cref="Microsoft.ML.Trainers.FastTreeBinaryClassifier"/>) in the pipeline.
+    /// 
+    /// Data can be analyzed at every step by inspecting the LearningPipeline object in VS.Net debugger.
+    /// <example>
```

Comment:
In [https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/xmldoc/example](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/xmldoc/example) it shows that `<example>` is not nested inside the `<summary>` section.  Instead, it is its own top-level section.